netmanager/src/views/containers/PageAccess.js

```diff
@@ -8,7 +8,7 @@ export const withPermission = (Component, requiredPermission) => {
     // Check if the user has the required permission
     const hasPermission =
       currentRole &&
-      currentRole.role_permissions.some(
+      currentRole?.role_permissions?.some(
```

Comment:
@OchiengPaul442 could this be the same chaining syntax? 
Won't we encounter some problems (mentioned earlier) in the older browsers or node/JS versions?